src/lib/format.ts

```diff
@@ -0,0 +1,85 @@
+import {
```

Comment:
This file feels verbose, but it should be useful more generally as we migrate to `Format`.